INFO:root:XWEAT started
INFO:root:Translating terms from en to es
INFO:root:Embeddings are in vec format
INFO:gensim.models.utils_any2vec:loading projection weights from ../data/embeddings/ft_embeddings.300.vec
INFO:gensim.models.utils_any2vec:loaded (52490, 300) matrix from ../data/embeddings/ft_embeddings.300.vec
INFO:root:Embeddings loaded
INFO:root:Running test
WARNING:root:Not in vocab de ella
WARNING:root:Not in vocab adici√≥n
INFO:root:Popped T2 26
INFO:root:Popped T2 14
INFO:root:Calculating p value ... 
INFO:root:Number of possible permutations: 924
INFO:root:(0.22765104, 0.81797534, 0.09199134199134198)
INFO:root:XWEAT started
Traceback (most recent call last):
  File "xweat/weat.py", line 694, in <module>
    train_weat(args)
  File "xweat/weat.py", line 624, in train_weat
    if args.test_number == 1:
AttributeError: 'NoneType' object has no attribute 'test_number'
